src/main/java/frc/robot/subsystems/Camera/Camera.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.Camera;
+
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonPipelineResult;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+public class Camera extends SubsystemBase {
+  PhotonCamera camera;
+
+  public Camera() {
+    this.camera = new PhotonCamera("slotheye");
+  }
+
+  /**
+   * An example method querying a boolean state of the subsystem (for example, a digital sensor).
+   *
+   * @return value of some boolean subsystem state, such as a digital sensor.
+   */
+  public boolean exampleCondition() {
+    // Query some boolean state, such as a digital sensor.
+    return false;
+  }
+
+  @Override
+  public void periodic() {
+    
+  }
+
+  @Override
+  public void simulationPeriodic() {
+    // This method will be called once per scheduler run during simulation
+  }
+
+  public double getAngle() {
+    PhotonPipelineResult result = camera.getLatestResult();
+    PhotonTrackedTarget target = result.getBestTarget();
+    Transform3d pose = target.getBestCameraToTarget();
+    Rotation3d rotation = pose.getRotation();
+    double angle = rotation.getAngle();
+    double xRot = rotation.getX();
+    System.out.println("Angle: " + angle*180/3.14);
```

Comment:
We do not need xRot, as the robot can't move by the roll reading from a gyro, or in this case a rotation around the X from the camera reading, so `rotation.getX()` would not be needed . Also can we instead print this to the SmartDashboard
Something like - `SmartDashboard.putNumber("Angle", angle*180/3.14)`